server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -455,8 +457,10 @@ func (r *statsRecorder) handleTask(ctx context.Context, task *recordStatsTask) {
 			ctx := usageutil.WithLocalServerLabels(ctx)
 
 			fullPath := path.Join(task.invocationJWT.id, cacheArtifactsBlobstorePath, uri.Path)
-			if err := persistArtifact(ctx, r.env, uri, fullPath); err != nil {
-				log.CtxError(ctx, err.Error())
+			if urlutil.GetDomain(uri.Hostname()) == build_buddy_url.Domain() {
```

Comment:
Should we be comparing this to cache_api_url instead?

Reply:
mmm, probably, good catch.